designer/apps/pages/tests/test_create_pages.py

```diff
@@ -112,9 +113,9 @@ def _create_program_page_data(self, branding=True, program_documents=True, exter
                 'external_program_website-INITIAL_FORMS': '0',
                 'external_program_website-0-display': True,
                 'external_program_website-0-header': ' '.join([word.capitalize() for word in fake.words(nb=3)]),
-                'external_program_website-0-description': "<ul>{}</ul>".format(
+                'external_program_website-0-description': rich_text("<ul>{}</ul>".format(
```

Comment:
@jmbowman, I have a concern related to this change on whether it would affect the existing pages on prod somehow. Kindly look into it and see if we need to do something more on this concern?

Reply:
Could we keep using the old rich text editor at first, and try upgrading it later separately?  https://docs.wagtail.io/en/v2.8/releases/2.0.html#hallo-js-customisations-are-unavailable-on-the-draftail-rich-text-editor

That may help limit the number of things we need to change all at once.

Reply:
Added option to use old text editor as default in settings as per suggestion.